packages/mgt-chat/src/components/ChatHeader/ChatHeader.tsx

```diff
@@ -29,11 +30,15 @@ interface ChatHeaderProps {
 const ChatHeader = ({ chatState }: ChatHeaderProps) => {
   const styles = useHeaderStyles();
   const commonStyles = useCommonHeaderStyles();
+  const chatWebUrl = chatState?.chat?.webUrl ?? 'https://teams.microsoft.com';
```

Comment:
I think we could link to v2 to leverage the latest improvements on the Teams client. This would be `https://teams.microsoft.com/v2`